provisioning/service/samples/GroupCertificateVerificationSample/GroupCertificateVerificationSample.csproj

```diff
@@ -0,0 +1,12 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="BouncyCastle.Crypto.dll" Version="1.8.1" />
```

Comment:
>Include [](start = 22, length = 7)

nit: not sure if we should use the one you listed or this one: [https://www.nuget.org/packages/BouncyCastle/](https://www.nuget.org/packages/BouncyCastle/)

the owner of this pkg is listed as Bouncy Castle Project Contributors

Reply:
Thanks @alextolp: I was actually looking into this right now. Changed to the official one.